src/ec/packages/ec-component-fact-figures/ecl-fact-figures.html.twig

```diff
@@ -54,7 +54,7 @@
   <div class="ecl-fact-figures__items">
   {% for item in items %}
     <div class="ecl-fact-figures__item">
-    {% if item.icon is defined and item.icon is not empty %}
+    {% if item.icon is defined and item.icon is not empty and display_icons == 1 %}
```

Comment:
This is a boolean, you can check for it's true value simply like this: `and display_icons`, @Joosthe

Reply:
Indeed I totally forgot to update readme and the template + i have set the code to local variables